hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -1240,6 +1240,15 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.namenode.access-control-enforcer-reporting-threshold-ms</name>
+  <value>1000</value>
+  <description>
+    If an external AccessControlEnforcer runs for a long time to check permission with the FSnamesystem lock,
+    we will print a WARN message when it happens. This sets how long must be run for logging to occur.
```

Comment:
Change `we will print a WARN message when it happens` to `print a WARN log message`.